index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Poetry</title>
+  </head>
+   <body style="background-color:lightblue;">
+  <body>
```

Comment:
2 notes here:
1. You have double `body` tags and should remove one.
1. All styles should be in your separate CSS document, not inline in your HTML.

In general, one of the benefits of thinking of HTML and CSS as separate languages is that they each have separate concerns: HTML is all about the meaning of your content and CSS is about the appearance of your content.

Writing styles this way, they are not reusable and you lose the benefits from your text editor autoformat and autocompletion for CSS.

```suggestion
  <body>
```